### ISSUE TITLE
Error in if (callr) { : argument is not interpretable as logical

### ISSUE THREAD

**Issue by milanmlft** [grimbough/bioc-actions#6]
Hi @grimbough,

Thanks a lot for the great actions, they really simplify my older GHA workflows a lot!

I'm currently running into an error on **Bioc-devel** though, during the `BiocCheck` step with the message

```
Error in if (callr) { : argument is not interpretable as logical
```

See for example https://github.com/milanmlft/msqrob2/actions/runs/3204812867/jobs/5236552594#step:7:76

I've been trying to reproduce this error both locally and using [my fork of your biocActionsExamples](https://github.com/milanmlft/biocActionsExamples/actions) but so far without much success...

However, I want to point out that in the biocActionsExamples workflow, the Bioconductor version used during `BiocCheck` is not correct for the `Bioc-devel` instances, e.g. in your latest run for `Bioc devel (macOS-11)`:

https://github.com/grimbough/biocActionsExamples/actions/runs/3205056371/jobs/5237096594#step:7:79

the bioc version indicated is actually 3.15 instead of 3.16... which is weird because in the `Set up R and Bioconductor` step, the version is correct. This might be a reason that the error is not showing up there. 
I thought that this might be due to some incorrect caching of the package dependencies, but I'm still exploring.

**Comment by milanmlft** [comment]
I haven't been able to reproduce this also not in my most recent runs where the Bioc version is correct.
So I will close this issue for now, as it might just have been a one-off event.

The problem with the inconsistent Bioc versions is still relevant though, but should be fixed in #7.

**Comment by grimbough** [comment]
Thanks a lot for the two issues.  

I think the {callr} problem was fixed by #5 and I made that available on a few hours before you created this issue.

Good spot on the Bioconductor version.  Hopefully your patch will fix the problem, and I'll try to put a test in my workflow to alert me if it happens again.  I'll merge that today.
